### ISSUE TITLE
Ground truth ply for Tanks and Temple intermediate test set

### ISSUE THREAD

**Issue by 320160040** [alibaba/cascade-stereo#26]
The official T&T dataset provides code for calculating F-Score along with ground truth plys for Training sets. The evaluation code requires ground truth data to compare with predicted data. However, I could not find ground truth data for intermediate test sets anywhere. So I wanted to ask if you created that by yourself by using some third-party software like COLMAP?

![image](https://user-images.githubusercontent.com/52123080/123197718-e08f8200-d4e6-11eb-9669-0899ae5180b2.png)

**Comment by tourlics** [comment]
> The official T&T dataset provides code for calculating F-Score along with ground truth plys for Training sets. The evaluation code requires ground truth data to compare with predicted data. However, I could not find ground truth data for intermediate test sets anywhere. So I wanted to ask if you created that by yourself by using some third-party software like COLMAP?
> 
> ![image](https://user-images.githubusercontent.com/52123080/123197718-e08f8200-d4e6-11eb-9669-0899ae5180b2.png)

Hello, it seems that the groundtruth of official T&T Testing dataset is not public, we can only submit our model to the online benchmark to get the F-score (like the table in this picture).

**Comment by wangchengze001** [comment]
how to change in code test.py，i can eval the T&T dataset

**Comment by zhao-you-fei** [comment]
@320160040 @wangchengze001 @tourlics @gxd1994 @alibaba-oss 
请教一下我在casmvsnet上测试T&T数据集没有报错,filter-method用的normal,为什么我的点云融合不成功或者说效果很差,ply_local文件夹下中间点云效果也不好,深度图效果也很差(如下图)
最后结果:
![image](https://user-images.githubusercontent.com/75787742/196881832-dd1db731-ca0d-4e95-b9b8-4ec7f7a39485.png)
ply_local文件夹下中间点云:
![image](https://user-images.githubusercontent.com/75787742/196881727-c5c56b27-6106-4ce1-bb9f-87117cae05b7.png)
深度图:
![image](https://user-images.githubusercontent.com/75787742/196881612-81ebd658-1f1a-4b93-9041-afe2feb0afec.png)
